### ISSUE TITLE
Make sure we're sending logs to Loggly

### ISSUE THREAD

**Issue by atavism** [getlantern/lantern-mobile#28]

**Comment by atavism** [comment]
Hey @myleshorton, I figured out why logs weren't being sent to Loggly:

```
I/GoLog   (24521): DEBUG flashlight.logging: logging.go:97 No logglyToken, not sending error 
logs to Loggly
```

We're setting the `logglyToken` variable using the `-X` Go linker option with `-ldflags`. Whenever we build the shared library using `gomobile bind -target=android -o=libflashlight.aar -ldflags="-X github.com/getlantern/flashlight/logging.logglyToken LOGGLY_TOKEN_FOR_MOBILE"` the token is never baked in/always undefined for whatever reason. I found a workaround setting it externally before we call `logging.Configure` for the meantime and the new build is able to write logs to the second Loggly instance we created.
